airmozilla/popcorn/urls.py

```diff
@@ -13,4 +13,7 @@
     url(r'^data/$',
         views.popcorn_data,
         name='popcorn_data'),
+    url(r'^vidly_webhook$',
```

Comment:
Try to avoid `_` underscore characters in the URL part. It's OK, and encouraged, in the function names.